coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
+    """
+    Escapes the given string with [c] pattern
+
+    :param string: String that is to be escaped with [ ]
+    :return:       Escaped string in which all the special glob characters
+                   such as ()[]!? are escaped.
+    """
+    output = ""
+    special_chars = "()[]|?*"
+    for char in string:
+        if char in special_chars:
+            output += "[" + char + "]"
+        else:
+            output += char
+    return output
+
```

Comment:
could you not edit your comment? It's very confusing for people following via emails and makes the history go bogus